layouts/team/single.html

```diff
@@ -0,0 +1,69 @@
+{{ define "main" }}
+
+<div class="container-single-team">
+    <div class="left">
+        <img src="/team/images/{{ $.Param "picture" }}">
+
+        <div class="presentation">
+            <div class="presentation__full-name-and-post">
+                <div class="fullname">
+                    <h1>{{ $.Params.firstname }} {{ $.Params.lastname }}</h1>
+                </div>
+
+                <div class="post">
+                    <p>{{ $.Params.position }}</p>
+                </div>
+            </div>
+
+            <div class="presentation__contact">
+                <h3><img class="logo" src="{{ "image/at-solid.svg" | relURL }}" alt="logo at">Email</h3>
+                <p>{{ $.Params.email }}</p>
+
+                <h3><img class="logo" src="{{ "image/phone-alt-solid.svg" | relURL }}" alt="logo de téléphone">Téléphone</h3>
+                <p>{{ $.Params.phone }}</p>
+            </div>
+        </div>
+
+    </div>
+
+    <hr>
+
+    <div class="right">
+        <div class="exp">
+            <h2><img class="logo" src="{{ "image/briefcase-solid.svg" | relURL }}" alt="cartable de travail">Expériences professionnelles</h2>
```

Comment:
"cartable de travail", :)
Pas utile selon moi de mettre du texte alternatif pour des images n'apportant pas d'information.
Je dirais même de supprimer les icones sur cette page, ça va alléger et aussi car n'apportant pas d'information et pouvant ne pas être aussi inclusif que l'on voudrait : attaché-case = homme.

Et sinon Expériences tout simplement, on est déjà dans un cadre "pro".
```suggestion
            <h2>Expériences</h2>
```

Reply:
C'est vrai que ça apporte pas grand chose la pour le coup les alternative d'image mais je pensais que c'était obligatoire du coup j'en ai quand même mis ^^
 ça marche , j'enlève tout les icônes

Reply:
Supprimer les images aussi de la PR ?